.env

```diff
@@ -0,0 +1,3 @@
+PORT=3000
+GOOGLE_APPLICATION_CREDENTIALS=
+SPREADSHEET_ID=
```

Comment:
I don't think we want to commit this file

Reply:
The `.env` file should _never_ be committed. See: https://www.npmjs.com/package/dotenv#FAQ

The main reason is that this file is the natural location for people to put credentials in, it's where they are supposed to go, after all. If we commit this then you create a _very_ easy vector for sensitive data to be exposed

I notice that the authors of the 'dotenv-flow' package think that [committing this file is okay](https://www.npmjs.com/package/dotenv-flow#files-under-version-control). In my opinion, they are wrong on this point and this makes me nervous about using this package at all. Maybe we should stick to https://www.npmjs.com/package/dotenv?

The Gateway project has [a good example](https://github.com/guardian/gateway/blob/main/docs/development.md#environment-variables) for using `env` files we could look to.

Just in terms of background information, the Guardian takes handling sensitive data seriously. The contents of things like env files, access keys, passwords, or _any_ reader data such as emails is all treated as sensitive so is not committed. We also make sure to not paste it in chat, emails, etc. Instead, we use an internal tool called Zero Bin for exchanging sensitive data.